tokio/src/runtime/threadpool/builder.rs

```diff
@@ -101,6 +102,36 @@ impl Builder {
         self
     }
 
+    /// Sets a callback to be triggered when a panic during a future bubbles up
```

Comment:
Just a nit: It's good to start the explanation with a single sentence followed by a more detailed explanation in a new paragraph.